Automattic/wp-calypso
# VideoPress User Intent: add a scrollable pane to the modal to support short screens

Related to # https://github.com/Automattic/greenhouse/issues/1843

## Proposed Changes

* create a scrollable area in the modal to prevent content from being hidden on short screens
* prevent the background from scrolling when the modal is visible

<img width="1401" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4081020/1750eac3-e6b8-4b06-8e57-ac5e15343c05">

## Testing Instructions



* `yarn && yarn start`
* `/setup/videopress`
* open each of the intents, they should look as expected
* the background should not scroll if you try scrolling
* shrink the height of your browser
* open each of the intents, you should be able to scroll to see the content
* the background page should not scroll when you reach the limits of the modal's scrolling

## Pre-merge Checklist
